shell/assets/styles/base/_mixins.scss

```diff
@@ -158,5 +158,6 @@
 
 @mixin focus-outline {
   // Focus for form like elements (not to be confused with basic :focus style)
-  outline: 2px solid var(--primary-keyboard-focus);
+  // we need to use !important because it needs to superseed other classes that might impact outlines
+  outline: 2px solid var(--primary-keyboard-focus) !important;
```

Comment:
Do we really need to use `!important`? We can usually apply styles by increasing specificity without the need for `!important`.

Reply:
took me a while to find out where the problem was but eventually found it. Removed the `!important`